### ISSUE TITLE
Finetuning for SO-101 Arm

### ISSUE THREAD

**Issue by dekelcohen** [Psi-Robot/DexGraspVLA#23]
Hi,

First, I was very impressed by your results.
I have a Robotic arm SO-101 (6 DOF) compatible with the LeRobot framework.
1) How much data do I need to finetune your checkpoint (pre-trained on different hardware) or should I start from scratch ? I can mask the unused dims.
2) Can I convert any LeRobot dataset (images+ proprio --> actions) to your format by using the pipeline models to create the mask ?

Thanks,
Dekel

**Comment by jcl2023** [comment]
If it can support so-arm101 with lerobot dataset for finetuning, it will have widely use.

Thanks,
